PixiEditor/Models/ImageManipulation/BitmapUtils.cs

```diff
@@ -106,6 +107,25 @@ public static WriteableBitmap GeneratePreviewBitmap(Document document, int maxPr
             return previewBitmap.Resize(width, height, WriteableBitmapExtensions.Interpolation.NearestNeighbor);
         }
 
+        public static WriteableBitmap GeneratePreviewBitmap(IEnumerable<SerializableLayer> layers, int width, int height, int maxPreviewWidth, int maxPreviewHeight)
```

Comment:
What did you change in this function?

Reply:
Made it use `SerializableLayer` instead of `Document`

Reply:
You can reuse a lot of code. DRY